modules/mapper-extras/src/main/java/org/elasticsearch/index/mapper/extras/RankFeatureFieldMapper.java

```diff
@@ -62,24 +64,37 @@ public static class Builder extends FieldMapper.Builder {
             true
         );
         private final Parameter<Map<String, String>> meta = Parameter.metaParam();
+        private final Parameter<Float> nullValue = new Parameter<>("null_value", false, () -> null,
+            (n, c, o) -> o == null ? null : ((Number)o).floatValue(), m -> ft(m).nullValue, XContentBuilder::field,
+            Objects::toString).acceptsNull();
 
         public Builder(String name) {
             super(name);
         }
 
         @Override
         protected Parameter<?>[] getParameters() {
-            return new Parameter<?>[] { positiveScoreImpact, meta };
+            return new Parameter<?>[] { positiveScoreImpact, nullValue, meta };
+        }
+
+        private void validateNullValue(Float nullValue) {
```

Comment:
Instead of this method, should we `.addValidator` function to `Parameter<Float> nullValue`.

Reply:
Done